app/models/app.rb

```diff
@@ -7,6 +7,7 @@ class App
   field :api_key
   field :github_repo
   field :bitbucket_repo
+  field :custom_backtrace_url
```

Comment:
The only remaining thing on my mind is this name. Which because it is a database field, we probably will never be able to change it. I'm trying to decide between this or 'backtrace_url' or 'backtrace_url_template' Have you considered any other names?

Reply:
My initial thought was custom_repo, but it's not correct. 
perhaps backtrace_url_template is a better term.  

When change the name to backtrace_url_template, IIcan change the method build_backtrace_url to backtrace_url. Which seems better.

Reply:
I like that!

Reply:
Maybe `custom_repo`? Just to be in sync with `github` and `bitbucket` `_repo`.

Reply:
I like the inclination toward consistency, but I think in this case these are too different to use the same name. One is a reference to a repo and the other is a template to use for a backtrace link. Maybe someday we can make all the backtrace links work using this new field even for GitHub and Bitbucket.